Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: drop Mac and Windows builds

We aren't going to build for those platforms anyway.

see also discussion in https://codeberg.org/openpgp-card/state/issues/5

Forwarded: not-needed

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,8 +46,6 @@
 [dependencies.keyring]
 version = "3"
 features = [
-    "apple-native",
-    "windows-native",
     "sync-secret-service",
 ]
 
